src/tokens/TokenDetailsMoreActions.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { RefObject } from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { AssetsEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import BottomSheet, { BottomSheetRefType } from 'src/components/BottomSheet'
+import Touchable from 'src/components/Touchable'
+import { Colors } from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { useTokenInfo } from 'src/tokens/hooks'
+import { TokenDetailsAction } from 'src/tokens/types'
+import { getTokenAnalyticsProps } from 'src/tokens/utils'
+
+export default function TokenDetailsMoreActions({
+  forwardedRef,
+  actions,
+  tokenId,
+}: {
+  forwardedRef: RefObject<BottomSheetRefType>
+  actions: TokenDetailsAction[]
+  tokenId: string
+}) {
+  const { t } = useTranslation()
+  const token = useTokenInfo(tokenId)
+  if (!token) throw new Error(`Token ${tokenId} not found`)
+
+  return (
+    <BottomSheet
+      forwardedRef={forwardedRef}
+      title={t('tokenDetails.moreActions')}
+      testId={'TokenDetailsMoreActions'}
```

Comment:
can we set a `titleStyle` based on the designs? I believe the bottom sheet defaults to one of the old font styles

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/4309/commits/d001e30e9196c60e10130c64debc8c1219ece3df!